recipes/openmm-atmmetaforce-plugin/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set name = "openmm-atmmetaforce-plugin" %}
+{% set version = "0.2.4.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Gallicchio-Lab/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: cf7975639c285ed0fab441c2ac36e3cbf6e499256a708240d4acfeb562939bae
+
+build:
+  number: 0
+  skip: true  # [not linux64]
+  skip: true  # [cuda_compiler_version in (undefined, 'None')]
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [linux]
+    - cmake
+
+  host:
+    - python
+    - openmm
+    - swig
+    - pthread-stubs
+    - lxml
+    # OpenCL ICD
+    - ocl-icd  # [linux]
+    - khronos-opencl-icd-loader  # [win or (osx and opencl_impl == 'khronos')]
+    
+  run:
+    - python
+    - openmm
+    # OpenCL ICD
+    - ocl-icd  # [linux]
+    - ocl-icd-system  # [linux]
+    - khronos-opencl-icd-loader  # [win or (osx and opencl_impl == 'khronos')]
+    - ocl_icd_wrapper_apple      # [osx and opencl_impl == 'khronos']
+
+test:
+  requires:
+    - pocl  # [unix]
+    - pip
+  imports:
+    - simtk
+    - simtk.openmm
```

```suggestion
    - openmm
```
There is a [deprecation warning](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=517904&view=logs&j=eb6ecf2b-0363-5c1a-baa7-2d771889d197&t=d8576026-8998-52ff-d027-1d631b0fb837&l=3148) for this test.

---

Done